### PR TITLE
micro: add page

### PR DESCRIPTION
**Pull request by kjullien**
Link to micro repository : https://github.com/zyedidia/micro
Micro is a nice little terminal based text editor with mouse capabilities and a very modern approach to keybinds, it is a perfect compromise between vim and nano.
It's a shame there isn't a tldr page on it yet even if I think it is somewhat of an overkill...

psa: I have nothing to do with micro or any of the devs of the project, i just really like it.

----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/micro.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# micro
+
+> Micro is a terminal-based text editor that aims to be easy to use and intuitive, while also taking advantage of the full capabilities of modern terminals.
```
Comment (pxgamer) [inline pages/common/micro.md]: Could you possibly shorten the description?

Perhaps something like:
> Micro is a modern and intuitive terminal-based text editor.

File: pages/common/micro.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+> You can use your keyboard, but also your mouse to navigate and/or select text.
+
+- Open a file:
+
+`micro {{file}}`
+
+- Cut the entire line:
+
+`<Ctrl>k`
```
Comment (pxgamer) [inline pages/common/micro.md]: Could you update the keyboard shortcuts to use the standard format?

e.g. `Ctrl + K`

See #1354 and #1356 for more details.

Comment (pxgamer) [inline pages/common/micro.md]: Could you update the keyboard shortcuts to use the standard format?

e.g. `Ctrl + K`

See #1354 and #1356 for more details.

**Comment by kjullien** [comment]
I changed 
- description
- shortcuts 
to match your feedback

File: pages/common/micro.md [hunk 3]
```diff
@@ -9,20 +9,20 @@
 
 - Cut the entire line:
 
-`<Ctrl>k`
+`<Ctrl> + K`
 
 - Search for a pattern in the file (press `<Ctrl>n`/`<Ctrl>p` to go to next/previous match):
```
Comment (pxgamer) [inline pages/common/micro.md]: Could you change the shortcuts here? And please could you also remove the `<` and `>` around the `Ctrl` for each shortcut.

Other than that, all looks fine to me.

**Comment by kjullien** [comment]
should be good now

**Pull request merged**